dl/src/main/scala/com/intel/analytics/bigdl/nn/Utils.scala

```diff
@@ -41,7 +41,7 @@ object Utils {
       if (v.isInstanceOf[Table]) {
         t1.update(k, zeroTableCopy(if (t1.contains(k)) t1(k) else T(), t2(k)))
       } else {
-        require(v.isInstanceOf[Tensor[T]], "Input can only consist of Tensor or Table")
+        require(v.isInstanceOf[Tensor[_]], "Input can only consist of Tensor or Table")
```

Comment:
Warns because the generic type is runtime-only and can't be used with isInstanceOf